pom.xml

```diff
@@ -1449,7 +1449,7 @@
         <generex.version>1.0.2</generex.version>
         <git-commit-id-plugin.version>4.9.10</git-commit-id-plugin.version>
         <groovy.version>3.0.9</groovy.version>
-        <guava.version>31.1-jre</guava.version>
+        <guava.version>32.0.1-jre</guava.version>
```

Comment:
what about updating the version to https://mvnrepository.com/artifact/com.google.guava/guava/32.1.3-jre instead?

Reply:
Thanks for the review, yes we can upgrade the version to 32.1.3 , build is successfull and tests as well.